bundle/metadata/annotations.yaml

```diff
@@ -4,8 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: splunk-operator
-  operators.operatorframework.io.bundle.channel.default.v1: "stable"
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.18.1
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
I am still not sure about this change. @pdhanoya-splunk or @sgontla can you guys please confirm if this is correct or if we need to revert it?

Reply:
I think we should revert it.